src/Resources/views/Ckeditor/browser.html.twig

```diff
@@ -77,57 +77,63 @@ file that was distributed with this source code.
 
     <div class="col-xs-12 col-md-9">
         <div class="box box-primary">
-            <div class="box-body {% if admin.datagrid.results|length > 0 %}table-responsive no-padding{% endif %}">
+            <div class="box-body {% if datagrid.results|length > 0 %}table-responsive no-padding{% endif %}">
```

Comment:
Why admin.datagrid doesn't work ? This seems pretty weird since, `admin` is used multiple times before.

Reply:
I also feel like a lot more code should be taken back from https://github.com/sonata-project/SonataAdminBundle/blob/4.x/src/Resources/views/CRUD/base_list.html.twig#L45-L105

Reply:
Does the ORM admin (injected here as admin) have sortparameters, datagrid and so on ?

I'd suggest setting up a project where you use 
[SonataDoctrineORMAdminBundle](https://github.com/sonata-project/SonataDoctrineORMAdminBundle)
NOT the abandoned SonataDoctrinePhpcrAdminBundle

and check what happens in this line:
admin.datagrid.sortparameters(field_description)

for example:
{% set sort_parameters      = admin.datagrid.sortparameters(field_description) %}

SonataDoctrinePhpcrAdminBundle 
https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle/blob/2.x/src/Model/ModelManager.php
HAS
public function getSortParameters(FieldDescriptionInterface $fieldDescription, DatagridInterface $datagrid)

while
https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/4.x/src/Model/ModelManager.php
DOES NOT HAVE

same applies to other datagrid/pager references
 
>Why admin.datagrid doesn't work ? This seems pretty weird since, admin is used multiple times before.
0. Check the above
1. You pass the datagrid directly to the template - why do this if you assume you have it in admin
2. I'll un-private test project skeleton that uses ORM Admin based setup when it fails. The only scenario where this might possibly work is IMO relying on abandoned PHPCR bundle.
3. If you rely on admin interface IMHO you can only relay on the things defined in the interface 
Sonata\AdminBundle\Model\ModelManagerInterface;
- no datagrid there.

Reply:
> Does the ORM admin (injected here as admin) have sortparameters, datagrid and so on ?

The admin is the same for all, the difference is the modelManager.

> I'd suggest setting up a project where you use
[SonataDoctrineORMAdminBundle](https://github.com/sonata-project/SonataDoctrineORMAdminBundle)
NOT the abandoned SonataDoctrinePhpcrAdminBundle

I'm using SonataDoctrineORMAdminBundle every day, never used SonataDoctrinePhpcrAdminBundle.

> You pass the datagrid directly to the template - why do this if you assume you have it in admin

The admin is automatically added to the template as a render parameter thanks to this method https://github.com/sonata-project/SonataAdminBundle/blob/4.x/src/Controller/CRUDController.php#L976

Sure, since the datagrid is passed, we can use it ; i'm not against it. But I'm more curious about which error you got first, because changing `admin.datagrid` to `datagrid` is not supposed to change/fix anything. There is something to study here.

Reply:
>The admin is the same for all, the difference is the modelManager.
Yep, sorry my mistake.

>but I'm more curious about which error you got first, because changing admin.datagrid to datagrid is not supposed to change/fix anything.

That is correct (it does not change anything).

I traced back to where I started,
The error was about  admin.datagrid.pager.nbresults
_Neither the property "nbresults" nor one of the methods "nbresults()", "getnbresults()"/"isnbresults()"/"hasnbresults()" or "__call()" exist and have public access in class "Sonata\DoctrineORMAdminBundle\Datagrid\Pager"._

The correct solution is to change this
datagrid.pager.countResults

I changed to datagrid from admin.datagrid as a first attempt to fix - just by looking on the variable stack in debugger and the wrong assumption it was caused by referring the wrong class.
My bad.

Reply:
also this was incorrect and original reason why I got rid of admin and changed to datagrid.sortparameters
admin.modelmanager.sortparameters(field_description, admin.datagrid)|merge(ckParameters)